Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -238,6 +243,7 @@ public class FileHandler: FileHandling {
 
     public func files(
         in path: Path.AbsolutePath,
+        directoryFilter: Set<Path.AbsolutePath>?,
```

Comment:
instead of another filter, what do you think about providing a closure:
```
public func files(
  in path: Path.AbsolutePath,
  filter: (URL) -> Bool = { _ true } 
)
```

Reply:
I changed it. Default arguments don't work for protocols. I kept the closure optional instead.